Northstar.Custom/mod/scripts/vscripts/sh_event_models.gnut

```diff
@@ -0,0 +1,25 @@
+global function EventModelsInit
```

Comment:
Why is this a shared file? Aren't things that server precaches automatically precached by clients?